R/pkg/R/functions.R

```diff
@@ -3803,3 +3803,89 @@ setMethod("repeat_string",
             jc <- callJStatic("org.apache.spark.sql.functions", "repeat", x@jc, numToInt(n))
             column(jc)
           })
+
+#' is_grouping
+#'
+#' Indicates whether a specified column in a GROUP BY list is aggregated or not,
+#' returns 1 for aggregated or 0 for not aggregated in the result set.
```

Comment:
is it not more intuitive if it is TRUE/FALSE? I guess this is the behavior in Scala?